client/components/Editor/Slides.jsx

```diff
@@ -169,72 +181,50 @@ class Slides extends React.Component {
                                 onChange={onChangeAddSlide}
                             />
                         </Grid.Row>
-                        {slides.map((slide, index) => (
-                            <Grid.Row
-                                key={slide.id}
-                                className="Slides-slide-sidebar-container"
-                            >
-                                <Card
-                                    className="Slides-slide-sidebar-card"
-                                    onClick={() =>
-                                        this.setState({
-                                            currentSlideIndex: index
-                                        })
-                                    }
+                        <Sortable onChange={onChangeSlideOrder}>
```

Comment:
Cool component!

Reply:
It only took me 4 tries...

![image](https://user-images.githubusercontent.com/27985/67601938-9b301580-f743-11e9-97e7-14651cf8c62d.png)